### ISSUE TITLE
e2e flake: Nodes Network when a minion node becomes unreachable [replication controller] 

### ISSUE THREAD

**Issue by gmarek** [kubernetes/kubernetes#17830]
Previous problem #13258 seems to have gone away. The new one is: 
```
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/resize_nodes.go:546 Expected error: <*errors.errorString | 0xc208128130>: { s: "timed out waiting for the condition", } timed out waiting for the condition not to have occurred
```

cc @wojtek-t @ixdy @brendandburns @quinton-hoole

**Comment by wojtek-t** [comment]
@gmarek - can you please paste the link to the example failure?

**Comment by gmarek** [comment]
http://kubekins.dls.corp.google.com/view/Flaky/job/kubernetes-e2e-gce-flaky/8136/

**Comment by wojtek-t** [comment]
This is duplicate of #14695

Closing this one in favor of #14695  (that will hopefully be fixed by #17939)
